media/audio-ui/build.gradle.kts

```
@@ -97,6 +97,7 @@ dependencies {
     implementation(projects.composeLayout)
     implementation(projects.composeMaterial)
     implementation(project(":composables"))
+    implementation(project(":media:audio-ui-model"))
```

can we use the typesafe API here?

```suggestion
    implementation(projects.media.audioUiModel)
```

---

And elsewhere